SU-SWS/react_paragraphs
# D8CORE-1829 Visual tweaks and API call adjustments

# READY FOR REVIEW

# Summary
- Reduced the API calls to occur only when editing the items. This speeds up initial loading because no API calls have to happen to render out the rows.
- Adjusted some padding on some elements
- Added a better loading component

# Review By (Date)
- 4/15

# Urgency
- low

# Steps to Test
1. checkout this branch
1. clear browser caches
1. verify all functionality still works
   - Create new paragraphs
   - drag those paragraphs
   - editing existing pages populates the fields with the accurate data.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
